internal/api/handlers/handler.go

```diff
@@ -171,3 +172,27 @@ func parseStateFilterQuery(
 	}
 	return stateEnum, nil
 }
+
+// parseTermsAcceptanceQuery parses the terms acceptance query and returns the address, public key, and terms accepted
+func parseTermsAcceptanceQuery(request *http.Request, btcNetParam *chaincfg.Params) (string, string, bool, *types.Error) {
```

Comment:
```suggestion
func parseTermsAcceptanceLoggingRequest(request *http.Request, btcNetParam *chaincfg.Params) (string, string, bool, *types.Error) {
```

It's not a query though right? It's a POST request.